SimGeneral/MixingModule/python/digitizers_Ph2_cfi.py

```diff
@@ -0,0 +1,127 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
this duplication is not allowed, please make the relevant changes in digitizers_cfi.py using eras